packages/solana/lib/src/helpers.dart

```diff
@@ -52,6 +52,16 @@ Future<String> findProgramAddress({
   throw const FormatException('cannot find program address with these seeds');
 }
 
+/// Create a program address for [programId] and [seeds]
+Future<String> createProgramAddress({
```

Comment:
How is this different from `findProgramAddress()`?

Reply:
findProgramAddress will find a nonce and add it to the seeds. This method is needed when you already know the nonce and want to get a program address

Reply:
findProgramAddress will find a nonce to append to the seeds. createProgramAddress is used when you already know the nonce.

Reply:
I see. What is the use case for this? Can you perhaps remove this change from this PR and create a different PR? It is unrelated to the other change. We are very interested in merging this, so if you could that would be great. Thanks.